launcher/src/components/UI/node-header/ServiceLinks.vue

```diff
@@ -46,15 +49,34 @@ export default {
       showMevboostWindow: false,
     };
   },
+
   computed: {
     ...mapState(useNodeHeader, {
       runningServices: "runningServices",
     }),
     ...mapState(useServices, {
       allServices: "allServices",
     }),
+    ...mapWritableState(useNodeHeader, {
+      testOperatorData: "testOperatorData",
+      pubkey: "pubkey",
+    }),
   },
+  mounted() {
+    //console.log(this.pubkey);
+  },
+
   methods: {
+    getKeys: async function () {
```

Comment:
don't execute these functions in `ServiceLinks.vue` execute them in `SsvModal.vue` where you actually need them then you also don't need the pinia store entries `pubkey` and `testOperatorData`